ansible_playbook.yml

```diff
@@ -0,0 +1,66 @@
+---
+- hosts: all
+  remote_user: vagrant
+  sudo: yes
+  tasks: 
+
+    - name: 'update'
+      apt:
+        update_cache: yes
+
+    - name: 'Looping installation of apt'
```

Comment:
Technically, It's not an `installation of apt` :P

Reply:
++